rust/tvm/Cargo.toml

```diff
@@ -40,6 +40,10 @@ tvm-macros = { version = "*", path = "../tvm-macros/" }
 paste = "0.1"
 mashup = "0.1"
 once_cell = "^1.3.1"
+pyo3 = { version = "0.11.1", optional = true }
```

Comment:
Nit: `^0.11.1`, to pick up new non-breaking changes? Or do we want `0.11.1` specifically?

Reply:
I believe that "^" is implicit, no? https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio

Reply:
TIL :)